src/content/protocols/chainlink-oracles.md

```diff
@@ -0,0 +1,112 @@
+---
+protocol: "Chainlink Price Feeds"
+website: "https://chain.link/"
+x: "https://x.com/chainlink"
+github: ["https://github.com/smartcontractkit"]
+defillama_slug: [""]
+chain: "Ethereum"
+type: "Oracles"
+logo: "/images/infrastructure-logos/chainlink-logo.png"
+stage: "I0"
+protocols: ["Morpho", "Aave v3", "Compound-V3", "Liquity"]
+reasons: []
+risks: ["L", "H", "L", "M", "L"]
+author: ["mmilien_"]
+submission_date: "2025-05-20"
+publish_date: "2025-05-20"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Chainlink offers real-time price feeds for several assets, which are broadly used in DeFi.
```

Comment:
Chainlink offers onchain price feeds for a variety of assets, which are broadly used in DeFi.